### PR TITLE
Change menu location

### PR DESCRIPTION
**Pull request by Brookwellreuber**
Moved Play Games Plugin menu location to be a Submenu under the Window menu on Unity's main menu bar.

### CODE REVIEW

File: source/PluginDev/Assets/GooglePlayGames/Editor/GPGSAndroidSetupUI.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ namespace GooglePlayGames {
     public class GPGSAndroidSetupUI : EditorWindow {
         string mAppId = "";
 
-        [MenuItem("Google Play Games/Android Setup...", false, 0)]
+        [MenuItem("Window/Google Play Games/Android Setup...", false, 0)]
```
Comment (claywilkinson) [inline source/PluginDev/Assets/GooglePlayGames/Editor/GPGSAndroidSetupUI.cs]: Can you remove the menuItem on line 31 also?

File: source/PluginDev/Assets/GooglePlayGames/Editor/GPGSIOSSetupUI.cs [hunk 2]
```diff
@@ -27,7 +27,7 @@ public class GPGSIOSSetupUI : EditorWindow {
         private string mClientId = "";
         private string mBundleId = "";
 
-        [MenuItem("Google Play Games/iOS Setup...", false, 1)]
```
Comment (claywilkinson) [inline source/PluginDev/Assets/GooglePlayGames/Editor/GPGSIOSSetupUI.cs]: On line 35 - can you remove that MenuItem?

**Comment by claywilkinson** [comment]
Thanks for this!  I made a couple minor comments - and can you update the README.md file where it references the menu items?

**Comment by Brookwellreuber** [comment]
Will do, I had a little trouble getting it to take the code changes in my repo and it looks like I missed the files there.

**Comment by Brookwellreuber** [comment]
With the setup menu items did you want them moved under the window? or under the window and then into the GPGS sub menu?

**Comment by claywilkinson** [comment]
I think Window->Google Play Games -> Setup ....   would be cleaner

**Comment by Brookwellreuber** [comment]
I've moved them to under GPG in window it looked like there was a duplicate menu entry so I removed one of the two set up menus.

**Comment by claywilkinson** [comment]
Thanks! The unitypackage needs to be rebuilt for everyone to get the goodness - I'll try to do this in the next couple of days.

**Pull request merged**